examples/storage/createjob.go

```diff
@@ -103,7 +95,7 @@ func main() {
 		log.Fatalf("ListJobs: %s", err)
 	}
 
-	fmt.Printf("Number of jobs: %d\n", jobs.ResultSetSize)
+	fmt.Printf("Result set size: %d\n", jobs.ResultSetSize)
```

Comment:
You can ignore this, I captured the chunked response in my proxy so I can create a separate issue for it.